### ISSUE TITLE
Why substract 1234 when processing the data

### ISSUE THREAD

**Issue by EternalConfession** [fulifeng/Temporal_Relational_Stock_Ranking#9]
I was very confused why substracting 1234 to the stock price.

Dose this operation have some realistic meanings?

Thanks

**Comment by fulifeng** [comment]
No, -1234 is just a special value which means the data is missing. You can replace it with any negative number that is large enough.

**Comment by EternalConfession** [comment]
Thanks for your reply.

Fuli Feng <notifications@github.com> 于2020年5月26日周二 上午1:02写道：
